src/modules/Classes.ts

```diff
@@ -0,0 +1,39 @@
+import Swup from '../Swup.js';
+
+export class Classes {
+	public swup: Swup;
+
+	swupClasses = ['to-', 'is-changing', 'is-rendering', 'is-popstate'];
+
+	constructor(swup: Swup) {
+		this.swup = swup;
+	}
+
+	get selectors(): string[] {
+		const { targets } = this.swup.context.transition;
+		return targets || ['html'];
+	}
+
+	get targets(): Element[] {
+		return Array.from(document.querySelectorAll(this.selectors.join(',')));
```

Comment:
We should rather iterate the array of containers here and execute `querySelector` for each of them. Otherwise, weird things could happen

Reply:
done